octue/resources/child.py

```diff
@@ -140,22 +187,18 @@ def ask_multiple(
         log_errors=True,
     ):
         """Ask the child multiple questions in parallel and wait for the answers. Each question should be provided as a
-        dictionary of `Child.ask` keyword arguments. If `raise_errors` is `True`, an error is raised and no answers are
-        returned if any of the individual questions raise an error; if it's `False`, answers are returned for all
-        successful questions while errors are returned unraised for any failed ones.
+        dictionary of `Child.ask` keyword arguments. The `raise_errors`, `max_retries`, `prevent_retries_when`, and
+        `log_errors` arguments have the same effect as in `Child.ask`, applied to all questions. However, if a question
+        specifies these arguments, the values provided by the question take precedence for that question.
```

Comment:
Suggest a slight rephrasing: `These values may be overridden on a per-question basis by specifying them in the question dictionary`